localstack/services/s3/s3_listener.py

```diff
@@ -1300,6 +1300,8 @@ def return_response(self, method, path, data, headers, response, request_handler
             convert_to_chunked_encoding(method, path, response)
 
             if headers.get('Accept-Encoding') == 'gzip':
+                if (isinstance(response._content, str)):
+                    response._content = response._content.encode()
                 response._content = gzip.compress(response._content)
```

Comment:
nitpick: Instead of the change in this PR, can we please utilize the existing `to_bytes(..)` util function (from `utils/common.py`) here:
```
response._content = gzip.compress(to_bytes(response._content))
```
(This function automatically converts between str/bytes representations, and also takes into account the subtle differences between Python 2.x and 3.x)

Reply:
fixed.